src/backends/git-gateway/implementation.js

```diff
@@ -43,7 +43,7 @@ export default class GitGateway extends GitHubBackend {
     AuthenticationPage.authClient = this.authClient;
   }
 
-  setUser() {
+  restoreUser() {
     const user = this.authClient && this.authClient.currentUser();
     if (!user) return Promise.reject();
     return this.authenticate(user);
```

Comment:
The result of what this function used to return was the localStorage `stored` value, now it would be the `this.authenticate` return value. I think this is what was originally intended, but not actually done, can someone else verify?

Reply:
Originally we weren't doing anything with the return value at all (only called that one time in `src/backends/backends.js`), so no worries there.

Reply:
@erquhart The difference was that before, the value stored in localStorage was returned, whereas now, the return value of this function is returned as the user.

Reply:
See https://github.com/netlify/netlify-cms/pull/559#issuecomment-334976455

Reply:
Right - but it didn't matter what was returned before because we weren't using that return value anywhere. That means there should be no backwards compatibility considerations.

Reply:
I'm talking about this line: https://github.com/netlify/netlify-cms/blob/master/src/backends/backend.js#L81. You can see there that we are returning the `stored` user object. VS now, at https://github.com/netlify/netlify-cms/blob/remembered-credentials-check/src/backends/backend.js#L86 we are actually returning the output of `restoreUser`.

Reply:
Ah, the return value of `currentUser` - I can't see why that would be an issue.